pkg/resource/deploy/source_eval.go

```diff
@@ -175,13 +185,14 @@ func (src *evalSource) Iterate(ctx context.Context, providers ProviderSource) (S
 }
 
 type evalSourceIterator struct {
-	mon         SourceResourceMonitor              // the resource monitor, per iterator.
-	src         *evalSource                        // the owning eval source object.
-	regChan     chan *registerResourceEvent        // the channel that contains resource registrations.
-	regOutChan  chan *registerResourceOutputsEvent // the channel that contains resource completions.
-	regReadChan chan *readResourceEvent            // the channel that contains read resource requests.
-	finChan     chan error                         // the channel that communicates completion.
-	done        bool                               // set to true when the evaluation is done.
+	loaderServer *plugin.GrpcServer
```

Comment:
I assume this is going to be used in a later commit/PR? If so, can we give it a doc comment? If not, feels like we can remove it?